test/spec-wasm2c-prefix.c

```diff
@@ -373,6 +373,7 @@ static void posix_signal_handler(int sig, siginfo_t* si, void* unused) {
 }
 
 static void posix_install_signal_handler(void) {
+#if !WASM_RT_NONCONFORMING_STACK_UNCHECKED
```

Comment:
Do we need these changes to spec-wasm2c-prefix.c?

Reply:
No strong opinion. This was done so that the rlbox wasm2c build in the CI could run tests in an environment using a setup that is as close to the real use as possible. Since the real use does not apply a stack exhaustion handler, we expect tests relying on stack-exhaustion-traps to fail, and this captures the expected behavior.

If you prefer not to do this, I can remove this (and also re-add the stack exhaustion tests in the rlbox wasm2c build in the CI)

Reply:
I can see both sides, but I guess on balance my preference (not that strong either) would be to avoid adding branches to the spec-test harness to accommodate a nonconforming use case; it doesn't feel like the complexity is worth it. No preference on whether you add those tests or not though -- dwyw.